packages/rum/src/types/sessionReplay.ts

```diff
@@ -34,6 +34,7 @@ export type CreationReason =
   | 'before_unload'
   | 'visibility_hidden'
   | 'page_frozen'
+  | 'page_hide'
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** Since we are not updating the schema, it could be nice to not update this file nether.
Someone could do a schema update on another topic and be bothered by that.